daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: docs
 title: "How-To: Invoke services using HTTP"
-linkTitle: "How-To: Invoke services"
+linkTitle: "How-To: Invoke services using HTTP"
```

Comment:
Please leave the link title short (this is what shows up in the left side tree). If we want to keep "HTTP" then maybe "How To: HTTP Invocation"

Reply:
Ok. Change to "How To: HTTP Invocation" and "How To: gRPC Invocation"

Reply:
Ok, changed to `How-To: Invoke with HTTP` and `How-To: Invoke with gRPC`